deployments/hadoop-yarn/ansible/roles/alertmanager/vars/main.yml

```diff
@@ -0,0 +1,6 @@
+---
+serviceName: "alertmanager"
+userId: "alertmanager"
+groupId: "alertmanager"
```

Comment:
The generic variable names work OK for a blog example, but they may cause unexpected conflicts as part of a larger production system. Suggest we add a prefix to them.
```suggestion
alertmanager_serviceName: "alertmanager"
alertmanager_userId: "alertmanager"
alertmanager_groupId: "alertmanager"
```

Reply:
I think these are specific to each role, as each role has a set of these variables names, which are used within the role and get the right value. Adding a prefix would require making modifications to other files within each role that these variables are referenced. Is this something urgent that you think should happen in this PR, or are we ok leaving it as is?

Reply:
As this is intended as a production service I think we should add the prefixes now. If in six month's time we see an error messages referring to `invalid userId` it will be hard to figure out what component the error applies to. An error message referring to `invalid alertmanager_userId` will be easier to track down.